applications/TheLostDiamond_BjornOmsland.md

```diff
@@ -0,0 +1,194 @@
+### Contact
+
+- **Contact Name:** Bjørn Omsland
+- **Contact Email:** bomsland@gmail.com
+- **Website:** thelostdiamond.io
+- **Time2Discover Website:** time2discover.no
+
+### Legal Structure
+- **Registered Legal Entity:** Time2Discover AS (Norway) https://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=925474746
+- **Registered Address:** Rosenholmveien 25, 1414 Trollåsen, Norway
+
+### Team Experience
+
+We are four creative outdoor enthusiasts from Norway with backgrounds in the travel industry, logistics, project management and programming.
+
+We have hosted several Lost Diamond events in Norway, utilizing the EOS-Mainnet, for more than 1,000 users to test and improve TheLostDiamond race.
+
+Our programmer (Bjørn Omsland) has twenty years experience as a full stack developer and four year experience as a smart contract developer on the EOS-blockchain. 
+
+
+### Team Code Repos
+
+- https://github.com/bjornomsland
+- https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+N/A
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/bjornomsland/
+- https://www.linkedin.com/in/ann-magrit-monhof-55785610/
+- https://www.linkedin.com/in/jorn-balslev/
+- https://www.linkedin.com/in/sivbalslev/
+
+## Development Status
+
+We have a working MVP that has been used for testing the application in several single-event races in Norway. https://cptblackbill.com/
+
+The smartcontract is located on the EOS-Mainnet account cptblackbill. The code is available at https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+The frontend is developed in NodeJs and hosted on Google Clouds. 
+
+Different single-event races can have different unique domain names. E.g. our biggest single-event race is “Discover Oslo Adventure Race”. We will host our third Discover Oslo Adventure Race on June 11th. https://opplevoslo.app The feedback from our Discover Oslo participants has been exceptionally good. 98% say they will participate in the upcoming Discover Oslo race.
+
+Development, testing and feedback from all the single-event races is the foundation for our main goal - launching The Lost Diamond 24/7 Global Adventure Race. The Lost Diamond 24/7 Global Adventure Race is ready for Test Marketing, where the goal is to find out how many people will participate in a pure crypto-based adventure race and see if we can obtain a 22% monthly growth rate. 
+
+
+## Development Roadmap
+
+**Marketing Roadmap**
+
+The market rollout strategy is divided into 24 monthly marketing campaigns. Each with a key result goal of 22% average growth rate.
+Milestone 1 covers the two first 24 monthly marketing campaigns. (Our understanding of the ENF grant framework is that we can only apply for $10,000 in the first request. If possible, we will apply again when Milestone 1 is accomplished within acceptable levels).
```

Comment:
Here it says Milestone 1 covers 24 months, below 2 months.  I suspect the latter is the truth but this should be fixed.

Reply:
Thanks for suggesting improvements on The Lost Diamond application.
I have:
- Added a definition for "participants": users that pay a $7 monthly subscription fee.
- Added "current baseline": 0 users (since we have not started marketing) 
- Defined explicit target number of users: Milestone 1 is to get a baseline of 100 participants paying a $7 dollar monthly subscription fee.
- Updated information about Milestone 1.